Common/ModelsDTO/UserDTO.cs

```diff
@@ -10,10 +10,10 @@ public class UserDTO
         public string LastName { get; set; }
         [Required]
         [EmailAddress(ErrorMessage = "Invalid Email Address")]
-        
         public string Email { get; set; }
         [Required]
-        [MinLength(6), MaxLength(16)]
+        [MinLength(8)]
+        [RegularExpression(@"^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%^&+=.,\-_!])([a-zA-Z0-9 @#$%^&+=*.,\-_!]){8,}$", ErrorMessage = "Password invalid.")]
```

Comment:
'Password is invalid' + put this string to Constants class

Reply:
Done